src/mono/nuget/Microsoft.NET.Sdk.Mono.Toolchain.Manifest/WorkloadManifest.targets

```diff
@@ -0,0 +1,46 @@
+<Project>
```

Comment:
If this is a shipping package then it needs the copyright section, i.e. https://github.com/dotnet/arcade/blob/cc512d28c9368336a2893172cbd45dc27960e304/src/Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk/src/build/Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk.props#L1